internal/utils/config.go

```diff
@@ -36,6 +36,21 @@ var (
 	VectorId      string
 	PoolerId      string
 
+	DbAliases          = []string{"db"}
+	KongAliases        = []string{"kong", "api.supabase.internal"}
```

Comment:
re: `api.supabase.internal`:

Personally I like this as a FQDN, feels more approachable for users. Also similar Docker's own DNS aliases, ie. `host.docker.internal`. Thoughts? Any downsides?

Reply:
Sounds good to me. Are we going to update the other services as well? ie. `db.supabase.internal` etc.

Reply:
Otherwise feel free to merge this PR as it is.

Reply:
Added `db.supabase.internal`, but leaving the rest until a use case arises (most are proxied through kong).